responsibleai/tests/explainer/test_explainer_manager.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) Microsoft Corporation
+# Licensed under the MIT License.
+
+import pytest
+
+from ..common_utils import create_iris_data, create_lightgbm_classifier
+
+from responsibleai.rai_insights import RAIInsights
+from responsibleai.exceptions import UserConfigValidationException
+
+
+class TestExplainerManager:
+    def test_explainer_manager_request_global_explanations(self):
+        X_train, X_test, y_train, y_test, feature_names, _ = \
+            create_iris_data()
+
+        model = create_lightgbm_classifier(X_train, y_train)
+        X_train['target'] = y_train
+        X_test['target'] = y_test
+
+        rai_insights = RAIInsights(
+            model=model,
+            train=X_train,
+            test=X_test.iloc[0:10],
+            target_column='target',
+            task_type='classification')
+        rai_insights.explainer.add()
+        rai_insights.compute()
+
+        global_explanations = \
+            rai_insights.explainer.request_explanations(
+                local=False, data=X_test.drop(['target'], axis=1).iloc[0:10])
+        assert global_explanations is not None
```

Comment:
nit: can these asserts be refactored out into a separate method and called in both tests?  It looks like we are validating the same properties in both test cases on global and local explanations.

Reply:
Done.